### PR TITLE
Removed all EgammaAnalysis packages except for ElectronTools

### PR DESCRIPTION
**Pull request by guitargeek**
#### PR description:

As I have seen on the egamma meeting at the last CMS week, one of the plans is to get rid of some old ID code. Why not start here with these old skims and IDs? Are there used for anything @Sam-Harper?

#### PR validation:

Code compiles and matrix tests pass.

### CODE REVIEW

File: Utilities/ReleaseScripts/scripts/git-publish [hunk 1]
```diff
@@ -405,9 +405,8 @@ CMSSW_CATEGORIES={"Operations": ["Configuration/Applications",
   "AnalysisDataFormats/SUSYBSMObjects", "AnalysisDataFormats/SiStripClusterInfo",
   "AnalysisDataFormats/TopObjects", "AnalysisDataFormats/TrackInfo",
   "DiffractiveForwardAnalysis/Configuration",
-  "DiffractiveForwardAnalysis/Skimming", "EgammaAnalysis/CSA07Skims",
-  "EgammaAnalysis/Configuration", "EgammaAnalysis/ElectronIDESSources",
-  "EgammaAnalysis/PhotonIDProducers", "ElectroWeakAnalysis/Configuration",
+  "DiffractiveForwardAnalysis/Skimming",
+  "ElectroWeakAnalysis/Configuration",
```
Comment (slava77) [inline Utilities/ReleaseScripts/scripts/git-publish]: I seem to recall that this file is supposed to contain all known packages from the past, not just the current release.

**Comment by Sam-Harper** [comment]
I'm totally fine with this :D  Thanks Jonas :)

**Comment by slava77** [comment]
@Sam-Harper 
please confirm that the removal is OK for the POG needs.

File: Utilities/ReleaseScripts/scripts/git-publish (hunk 1 above)
Comment (guitargeek) [reply Utilities/ReleaseScripts/scripts/git-publish]: Really? I did the same thing in another PR where I removed a package:
https://github.com/cms-sw/cmssw/pull/26186
Back then, @Dr15Jones was giving a +1 for that. So maybe I should also revert that change?

Comment (slava77) [reply Utilities/ReleaseScripts/scripts/git-publish]: nevermind, I may have misunderstood/confused with another file with a list of packages.
 This one is in the core signautre list anyways and will be confirmed.

**Comment by Dr15Jones** [comment]
please test

**Comment by santocch** [comment]
+1

**Comment by Dr15Jones** [comment]
+1
for Utilities/ReleaseScripts

**Comment by kpedro88** [comment]
+1

**Pull request merged**